pages/design-tokens/flex.md

```diff
@@ -126,7 +130,7 @@ Your context and coding style determine how you access USWDS flex tokens in code
         </td>
         <td data-title="Example">
           <span>
-            $theme-nav-column-flex: <code>4</code>
+            $theme-megamenu-columns: <code>4</code>
```

Comment:
Not sure if this is a suitable replacement, but was the only setting I could find that was pulled into `u-flex`. Please let me know if this feels wrong to include here and/or if you have a suitable replacement.